File: overview/src/main/java/cn/edu/nuc/overview/OverviewActivity.java
package cn.edu.nuc.overview;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.alibaba.android.arouter.facade.annotation.Route;

@Route(path = "/pbs_ov/1")
public class OverviewActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.ov_activity);
    }

}

File: overview/src/main/java/cn/edu/nuc/overview/OverviewActivity.kt
package cn.edu.nuc.overview

import android.support.v4.app.Fragment
import cn.edu.nuc.library.base.FragmentContainerActivity
import com.alibaba.android.arouter.facade.annotation.Route

@Route(path = "/pbs_ov/1")
class OverviewActivity : FragmentContainerActivity() {

    override val layoutResId = R.layout.ov_activity

    override val fragmentContainerId: Int = R.id.ov_fragment_container

    override fun createFragment(): Fragment = OverviewFragment.newInstance()

}
